src/components/FeatureViews/Dashboard/Dashboard.js

```diff
@@ -38,21 +39,59 @@ const useStyles = makeStyles((theme) => ({
 
 const Dashboard = () => {
     const classes = useStyles();
-    const user = useSelector(({auth}) => auth) || [];
+    const {email} = useSelector(({auth}) => auth) || [];
+    const emailQuery = { "email": email }
+    const USER_QUERY = gql`query userQuery($email: String="") {
+        accountSearch(query: $email) {
+          results {
+            ... on AdminType {
+              user {
+                email
+                firstName
+                id
+              }
+            }
+          }
+        }
+      }
+      
+      
+      
+      `;
+      
+    const user = useQuery(USER_QUERY, { 
+        variables: emailQuery, 
+    })
+
```

Comment:
The useQuery hook, gives us an object with 3 attributes: {data, loading, error}. 

Whenever you use useQuery, always make sure loading = true before using data. 

You'll want to handle that on this line by doing this:
```
if( loading ) { //loading component } 
```